manifests/server/schema.pp

```diff
@@ -8,22 +8,30 @@
 # @param owner Sets the default owner of the schema.
 # @param schema Sets the name of the schema.
 # @param connect_settings Specifies a hash of environment variables used when connecting to a remote server.
+# @param user Sets the OS user to run psql
+# @param group Sets the OS group to run psql
+# @param psql_path Sets path to psql command
+# @param module_workdir
+#   Specifies working directory under which the psql command should be executed.
+#   May need to specify if '/tmp' is on volume mounted with noexec option.
+# @param version Deprecated. Sets PostgreSQL version
```

Comment:
is that actually deprecated? can we remove? all those changes should be moved into its own PR as well